fsgs/FSGSDirectories.py

```diff
@@ -12,6 +12,45 @@
 
 
 class FSGSDirectories(object):
+
+    CACHE_DIR = 'cache_dir'
+    CDROMS_DIR = 'cdroms_dir'
+    CONTROLLERS_DIR = 'controllers_dir'
+    COVERS_DIR = 'Covers'
+    DATA_DIR = 'data_dir'
+    DATABASES_DIR = 'databases_dir'
+    DOWNLOAD_DIR = 'download_dir'
+    FLOPPIES_DIR = 'floppier_dir'
+    KICKSTARTS_DIR = 'kickstarts_dir'
+    HARD_DRIVES_DIR = 'hard_drives_dir'
+    IMAGES_DIR = 'images_dir'
+    LOGS_DIR = 'logs_dir'
+    PLUGINS_DIR = 'plugins_dir'
+    SAVE_STATES_DIR = 'save_states_dir'
+    SCREENSHOTS_DIR = 'screenshots_dir'
+    THEMES_DIR = 'themes_dir'
+    TITLES_DIR = 'titles_dir'
+    # TODO: grab these names from LauncherConfig.get("name_dir")
+    DIRNAME_TO_REALPATH = {
+        CDROMS_DIR: 'CD-ROMs',
+        CACHE_DIR: 'Cache',
+        CONTROLLERS_DIR: 'Controllers',
+        COVERS_DIR: 'Covers',
+        DATA_DIR: 'Data',
+        DATABASES_DIR: 'Databases',
+        DOWNLOAD_DIR: 'Downloads',
+        FLOPPIES_DIR: 'Floppies',
+        KICKSTARTS_DIR: 'Kickstarts',
+        HARD_DRIVES_DIR: 'Hard Drives',
+        IMAGES_DIR: 'Images',
+        LOGS_DIR: 'Logs',
+        PLUGINS_DIR: 'Plugins',
+        SAVE_STATES_DIR: 'Save States',
+        SCREENSHOTS_DIR: 'Screenshots',
+        THEMES_DIR: 'Themes',
+        TITLES_DIR: 'Titles',
```

Comment:
with this list I think you see where I'm going: most if not all the methods in this class do the same thing: check path, create dir. So I think that it can be refactored with a single `get_dir(dir_name)`.

This list actually made me notice some inconsistencies, see other comments down below